ChopChop/Models/Cooking/SessionRecipeStepGraph.swift

```diff
@@ -46,10 +46,25 @@ class SessionRecipeStepGraph {
 
         self.actionTimeTracker = actionTimeTracker
         self.graph = sessionGraph
+
+        updateCompletableNodes()
+    }
+
+    func getTopologicallySortedNodes() -> [SessionRecipeStepNode] {
```

Comment:
Should we make this and getNodeLayers computed properties instead? (including the current methods in `DirectedAcyclicGraph` too)

Reply:
Yes I can do that.